src/pose-sensor.js

```diff
@@ -0,0 +1,169 @@
+/*
+ * Copyright 2015 Google Inc. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import FusionPoseSensor from './sensor-fusion/fusion-pose-sensor.js';
+import { Vector3, Quaternion } from './math-util.js';
+
+// Frequency which the Sensors will attempt to fire their
+// `reading` functions at. Use 16.6ms to achieve 60FPS, since
+// we generally can't get higher without native VR hardware
+const SENSOR_FREQUENCY = 16.6;
+
+const X_AXIS = new Vector3(1, 0, 0);
+const Z_AXIS = new Vector3(0, 0, 1);
+
+let orientation = {};
+if (screen.orientation) {
+  orientation = screen.orientation;
+} else if (screen.msOrientation) {
+  orientation = screen.msOrientation;
+} else {
+  Object.defineProperty(orientation, "angle", {
+    get: () => { return (window.orientation || 0) }
+  });
+}
+
+// Quaternion to rotate from sensor coordinates to WebVR coordinates
+const SENSOR_TO_VR = new Quaternion();
+SENSOR_TO_VR.setFromAxisAngle(X_AXIS, -Math.PI / 2);
+SENSOR_TO_VR.multiply(new Quaternion().setFromAxisAngle(Z_AXIS, Math.PI / 2));
```

Comment:
Is there a better way to transform sensor coordinates to WebVR coordinates other than this?